### ISSUE TITLE
🐛 [Bug] Browser support

### ISSUE THREAD

**Issue by Tirokk** [Human-Connection-Community/Human-Connection-New-20201003#2288]
> [<img alt="datenbrei" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/datenbrei) **Authored by [datenbrei](https://github.com/datenbrei)**
_<time datetime="2019-11-19T12:52:26Z" title="Tuesday, November 19th 2019, 1:52:26 pm +01:00">Nov 19, 2019</time>_
_Closed <time datetime="2020-02-17T15:35:01Z" title="Monday, February 17th 2020, 4:35:01 pm +01:00">Feb 17, 2020</time>_
---

## :bug: Bugreport
There are some browsers, like edge or older Safari browsers, which are not supported.

### Steps to reproduce the behavior
1. Try to login
2. See, that it does not work

### Expected behavior
Any browser should ether work or one should get a warning, that it is not supported - and what you could possibly do (take another browser)

### Version & Environment
- [x] support latest Edge version
- [ ] support IE from version .. ? No! ;-)

### Additional context
@datenbrei will add more specific information, which browsers have problems.

IE is currently failing on the login page because of missing `babel` settings in the styleguide. This could be fixed by either releasing a new version of the styleguide with updated settings or by completing the migration of the styleguide.

The implementation of the masonry layout is also not optimized for IE – this needs to be tested and might require a CSS fallback.

**Comment by Tirokk** [comment]
> [<img alt="cypress[bot]" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/apps/cypress) **Authored by [cypress[bot]](https://github.com/apps/cypress)**
_<time datetime="2019-11-19T15:27:05Z" title="Tuesday, November 19th 2019, 4:27:05 pm +01:00">Nov 19, 2019</time>_

---



<p align="center">
  <br />
  <img src="https://assets.cypress.io/github-pr-comment-icons/cypress-logo.png" height="32" />
</p>

---

### Test summary
<a href="https://dashboard.cypress.io/projects/qa7fe2/runs/2402/specs"><img src="https://assets.cypress.io/github-pr-comment-icons/green-check-mark.png" width="14" height="14" title="Passed" /></a> <b title="Passed">48</b>  &bull; <a href="https://dashboard.cypress.io/projects/qa7fe2/runs/2402/failures"><img src="https://assets.cypress.io/github-pr-comment-icons/red-x.png" width="14" height="14" title="Failed" /></a> <b title="Failed">0</b>  &bull; <img src="https://assets.cypress.io/github-pr-comment-icons/gray-circle.png" width="14" height="14" title="Tests that did not run due to a developer annotating a test with .skip" /> <b title="Pending">0</b> &bull; <img src="https://assets.cypress.io/github-pr-comment-icons/yellow-triangle.png" width="14" height="14" title="Tests that did not run due to a failure in a mocha hook" /> <b title="Skipped">0</b>

---

### Run details
| | |
| --- | --- |
| **Project** | Human-Connection |
| **Status** | Passed |
| **Commit** | 1dc68cbae8 |
| **Started** | Nov 27, 2019 9:09 AM |
| **Ended** | Nov 27, 2019 9:23 AM |
| **Duration** | 13:55 <a href="https://on.cypress.io/parallelization" title="Enable parallelization to reduce run duration">💡</a> |
| **OS** | Linux Ubuntu Linux - 16.04 |
| **Browser** | Chromium 78 <img src="https://cdnjs.cloudflare.com/ajax/libs/browser-logos/48.0.4/chromium/chromium_32x32.png" width="16" height="16" /> |

[View run in Cypress Dashboard ➡️](https://dashboard.cypress.io/projects/qa7fe2/runs/2402/specs)



---

This comment has been generated by **cypress-bot** as a result of this project's GitHub integration settings. You can manage this integration in [this project's settings in the Cypress Dashboard](https://dashboard.cypress.io/projects/qa7fe2/settings)

**Comment by Tirokk** [comment]
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2019-11-20T13:00:57Z" title="Wednesday, November 20th 2019, 2:00:57 pm +01:00">Nov 20, 2019</time>_

---


I spent a lot of time today figuring out how to make the global `BaseComponents` play nicely with our jest tests (the problem was that `jest` doesn't use `webpack` and therefore cannot deal with `require.context`)...

During my research I also found out that it is possible to set `localVue` as a global variable in a single `testSetup` file and then use this variable in all test files without the need to configure it over and over again – as we did before.

I thought about implementing this in a separate PR because this one is getting out of hand and will be a nightmare to review – but then I would have had to import `BaseComponents` in all the test files so they would have been changed anyway. Sorry! 🤷‍♀
